### DISCUSSION TITLE
Config production ec2

### DISCUSSION THREAD

**Discussion by kusmin** [chaskiq/chaskiq#985] [General]
First I would like to thank you for the fantastic app that is chaskiq!

Hi, I tried to deploy chaskiq on EC2, but I get an error when entering the home - https://upsocial.app.br/

![image](https://github.com/chaskiq/chaskiq/assets/28716255/d40fefb0-e82a-489a-aec0-0af8842416d5)

After installing the app, I ran the following commands in the container:

bundle exec rails db:migrate 
bundle exec rails db:seed 
bundle exec rails admin_generator

**Comment by michelson** [comment]
use env var: `TLD_LENGTH=2 ` as your domain has two levels
